subcell_analysis/compression_analysis.py

```diff
@@ -19,3 +18,106 @@ def asymmetry(fibers_df):
         #print(diff[index])
     print(len(diff))
     #write 
+
+
+def get_axis_distances_and_projections(fiber_points):
```

Comment:
would be cool to type these too, so this one would be 
`def get_axis_distances_and_projections(fiber_points: np.ndarray) -> Tuple[np.ndarry,np.ndarry,np.ndarry]:`